lib/_http_client.js

```diff
@@ -413,9 +413,13 @@ function socketCloseListener() {
   const res = req.res;
   if (res) {
     // Socket closed before we emitted 'end' below.
+    // TOOD(ronag): res.destroy(err)
     if (!res.complete) {
       res.aborted = true;
       res.emit('aborted');
+      if (res.listenerCount('error') > 0) {
+        res.emit('error', connResetException('socket hang up'));
```

Comment:
Doesn't this change the meaning of the "socket hang up" error? If I'm not work it was only emitted if no `'response'` event was emitted.

Reply:
changed to `'aborted'`